content/conduct/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Code of Conduct"
+draft: false
+weight: 110
+---
+
```

Comment:
Right here let's add this
```

- [Our pledge](#our-pledge)
- [Our Standards and Values](#our-standards-and-values)
- [Our Responsibilities](#our-responsibilities)
- [Scope](#scope)
- [Enforcement](#enforcement)
- [Attribution](#attribution)
```
and a new line as a clickable table of content

Reply:
On it

Reply:
Added, though I'm guessing there's a better way. Can you take a look?

Reply:
LGTM!